src/components/common/SearchComponent/index.tsx

```diff
@@ -0,0 +1,128 @@
+import styled from "styled-components";
+import { FC } from "react";
+import { colors, fontFamilies, sizes } from "../../../variables";
+import SearchBoxComp from "../searchBox";
+import LocationInput from "../locationInput";
+import DateInput from "../dateInput";
+import searchBoxBg from "./../../..//assets/common/foodSearchBox/searchbox_bg.png";
+
+interface foodSearchProps {
+  pageType: string;
+}
+
+const Foodsearch: FC<foodSearchProps> = ({ pageType }) => {
+  return (
+    <>
+      {pageType === "home" ? (
+        <WrapperHome className={pageType}>
+          <SearchBoxComp className={pageType} />
+          <LocAndDate className={pageType}>
+            <Locationholder className={pageType}>
+              <LocationInput className={pageType} />
+            </Locationholder>
+            <Dateholder className={pageType}>
+              <DateInput className={pageType} />
+            </Dateholder>
+          </LocAndDate>
+        </WrapperHome>
+      ) : (
+        <Wrapper>
+          <ContentContainer>
+            <DeliveryAsap>Delivery ASAP</DeliveryAsap>
```

Comment:
Move hardcoded string

Reply:
Moved hardcoded string to constants.json